### PR TITLE
Updating to core 1685

### PR DESCRIPTION
**Pull request by andreasohlund**
Removed a test that checked that transports could be configured since it doesn't belong here

### CODE REVIEW

**Comment by andreasohlund** [comment]
@Particular/nservicebus-maintainers please review

File: src/NServiceBus.Hosting.Tests/GetEndpointConfigurationType_Tests.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    using System.Collections.Generic;
+    using System.Configuration;
+    using System.Reflection;
+    using NServiceBus;
+    using NServiceBus.Hosting.Helpers;
+    using NServiceBus.Hosting.Windows;
+    using NUnit.Framework;
+
+    [TestFixture]
+    class GetEndpointConfigurationType_Tests
```
Comment (timbussmann) [inline src/NServiceBus.Hosting.Tests/GetEndpointConfigurationType_Tests.cs]: Also rename this class + file?

File: src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+namespace EndpointTypeDeterminerTests
+{
+    using NServiceBus;
+
+    class MyEndpointConfig : IConfigureThisEndpoint
+    {
+        public void Customize(EndpointConfiguration configuration)
+        {
+
```
Comment (timbussmann) [inline src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs]: :fire:

File: src/NServiceBus.Hosting.Windows/GenericHost.cs [hunk 3]
```diff
@@ -42,15 +41,12 @@ public GenericHost(IConfigureThisEndpoint specifier, string[] args, List<Type> d
...
-        /// <summary>
-        ///     Creates and starts the bus as per the configuration
-        /// </summary>
         public void Start()
         {
             try
             {
                 var startableEndpoint = PerformConfiguration().Result;
-                endpoint =  startableEndpoint.Start().Result;
+                endpoint = startableEndpoint.Start().Result;
```
Comment (timbussmann) [inline src/NServiceBus.Hosting.Windows/GenericHost.cs]: should we use `GetAwaiter().GetResult()` here?

Comment (danielmarbach) [inline src/NServiceBus.Hosting.Windows/GenericHost.cs]: Yes! I would like us to apply that pattern at the outermost level. So in that case if possible we should make Start and Stop async and synchronize higher up the call stack

File: src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs (hunk 2 above)
Comment (andreasohlund) [inline src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs]: It's actually used, see https://github.com/Particular/NServiceBus.Host/pull/57/files/3c1915c978913afc115aacb863ce4248cd738449#diff-f034fb2a8d52cb17d06a8e5992fd4d10R4

File: src/NServiceBus.Hosting.Windows/GenericHost.cs [hunk 4]
```diff
@@ -5,18 +5,17 @@ namespace NServiceBus
...
     using System.Reflection;
     using System.Threading.Tasks;
-    using Hosting.Helpers;
-    using Hosting.Profiles;
-    using Logging;
     using NServiceBus.Configuration.AdvanceExtensibility;
-
+    using NServiceBus.Hosting.Helpers;
+    using NServiceBus.Hosting.Profiles;
+    using NServiceBus.Logging;
```
Comment (andreasohlund) [inline src/NServiceBus.Hosting.Windows/GenericHost.cs]: Stupid formatting adding this back, will fix

**Comment by andreasohlund** [comment]
Review suggestions implemented

**Comment by danielmarbach** [comment]
Youd didn't address mine

**Comment by andreasohlund** [comment]
>Youd didn't address mine

Was just updating to latest core, can we deal with the async change in a different pull?

**Comment by danielmarbach** [comment]
Fair enough. As long as it gets done :)

**Comment by andreasohlund** [comment]
>Fair enough. As long as it gets done :)

Raise an issue so that we don't forget?

**Comment by danielmarbach** [comment]
https://github.com/Particular/NServiceBus.Host/issues/61

File: src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs (hunk 2 above)
Comment (timbussmann) [inline src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs]: @andreasohlund I meant the empty line...

Comment (andreasohlund) [inline src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs]: Ah, got it, thanks!

**Pull request merged**